### ISSUE TITLE
default icon for an app

### ISSUE THREAD

**Issue by lduboeuf** [ubports/unity8#210]
In case of resource load failure, can we provide a default icon as it was the case on Stable ?

**Comment by lduboeuf** [comment]
![screenshot20191021_234139879](https://user-images.githubusercontent.com/11663835/67267231-285c3b80-f4b2-11e9-9ad8-35a2236106ec.png)

**Comment by cibersheep** [comment]
There's an empty icon we use in `clickable` that could be used:
https://gitlab.com/clickable/ut-app-meta-template/blob/master/%7B%7Bcookiecutter%5B'App%20Name'%5D%7D%7D/assets/logo.svg

**Comment by lduboeuf** [comment]
yes, noticed it exist also in ubuntu-theme / or ui toolkit . But don't know if that change needs to be done on QML side or in model ( i can't build unity8 locally so can't test c++ code)

**Comment by lduboeuf** [comment]
for reference : https://github.com/ubports/unity8/pull/216

works need to be done on `ubuntu-app-launch` and theme
